include/ajax.tickets.php

```diff
@@ -565,7 +565,7 @@ function refer($tid, $target=null) {
                             if ($v[0] == '-')
                                 $remove[] = substr($v, 1);
                         if (count($remove)) {
-                            $num = $ticket->thread->referrals
+                            $num = $ticket->getThread()->referrals
```

Comment:
I don't think this helps at all - since getThread will return a null. Plenty of same instances in class.ticket.php.